src/components/04-list/list-plain-text/list-plain-text.njk

```diff
@@ -1,5 +1,5 @@
 <ul class="rvt-plain-list">
```

Comment:
I'm wondering if it might be better to rewrite the `list-plain-text` to be a variant of `list-unordered` since the only difference is the presence of the class `.rvt-plain-list`.

Reply:
That's a good idea. I can include the `inline` list variant within that as well, since that was left out in the list component refactor. That can be in a separate branch.